Automattic/wp-calypso
# TinyMCE: serve skin files from public directory in Calypso.

As a followup to #24782, as discussed in pMz3w-8Ak-p2, this adds the lightgrey theme to the public directory in Calypso, and updates the `skin_url` paths to point to the newly added theme.

## To test

Verify that the editor is not loading any TinyMCE-related files from WP.com-global files like `//s1.wp.com/wp-includes/js/...`, they should instead be loaded from Calypso itself:

![2018-05-09t22 34 12-0500](https://user-images.githubusercontent.com/227022/39850895-6c60e0f4-53d9-11e8-8dbb-4de6b5abba6d.png)

Verify that the editor still works and appears correctly styled.

Verify that the same approach works in the TinyMCE fields for the WooCommerce extension.
